src/dynsight/_internal/soapify/saponify.py

```diff
@@ -21,7 +21,7 @@ def saponify_trajectory(
     selection: str = "all",
     soap_respectpbc: bool = True,
     n_core: int = 1,
-    **soapkwargs: Any,
+    centers: str = "all",
```

Comment:
What is an example of another string? How do the user define this? IS there a limit of options?

Reply:
Maybe add a link to the mdnalysis selection docs?